[rocprofiler-compute] Update Documentation for Strix Halo (#4794)
ROCm/rocm-systems: projects/rocprofiler-compute/docs/conceptual/rdna/gl1-cache.rst

.. meta::

>    :description: ROCm Compute Profiler RDNA3.5 GL1C (L1) metrics

Reviewer: ```suggestion
   :description: Learn about the GL1C (L1) metrics in ROCm Compute Profiler, including utilization, request statistics, cache performance, and stalls on RDNA 3.5 (gfx1151).
```

>    :description: Learn about the GL1C (L1) metrics in ROCm Compute Profiler, including utilization, request statistics, cache performance, and stalls on RDNA 3.5 (gfx1151).

Reviewer 2: The GL1C != L1 in lots of documentation. It is a global L1 cache but not a generic L1 cache (it is too vague and could be mistaken for the vL1d, sL1D, L1 instr cache). I think we should remove the brackets or put "(GL1)" instead of "(L1)"

Reviewer 2: Apply same idea for rest of the file below

Reviewer 3: removed in commit "c7fabbae90598be84ccaf5441ea6973c8d91a6b1"

   :keywords: ROCm Compute Profiler, RDNA, gfx1151, GL1C, GL1

.. _rdna-gl1:

==========
GL1 (GL1C)
==========

> GL1C is the shared L1 cache layer on gfx1151, fed by TCP (vector L0).

Reviewer 2: I think we should reword this first sentence because it is using L0 and L1 too freely in combination with the blocks, also not sure about "vector L0".

TCP = vL1D (vector L1 data cache) = GL0 (L0 global cache) in gfx10+

Shader array cache (for all CU!) = GL1 = but also 'L0' is used for per-CU cache!!

> GL1C is the shared L1 cache layer on gfx1151, fed by TCP (vector L0).

Reviewer: ```suggestion
GL1C is the shared L1 cache layer on gfx1151, supplied by the TCP (vector L0).
```

For TCP panels and Memory Chart rows through the TCP-GL1 boundary, see

> :doc:`tcp-cache`. The GL1C to GL2 path is under :doc:`gl2-cache`.

Reviewer: ```suggestion
:doc:`tcp-cache`. The GL1C to GL2 path is described in :doc:`gl2-cache`.
```

GL1C (panel)
============

> 

Reviewer: This page also contains details on the Memory chart: (GL1C (L1), need an introductory text in both sections to have a clear distinction between GL1C (panel) and Memory chart: GL1C (L1).

Reviewer 3: will be worked later with separate PR

> GL1C Utilization

Reviewer: ```suggestion
GL1C utilization
```

----------------

.. tab-set::

   .. tab-item:: RDNA 3.5 (gfx1151)
      :selected:

      .. jinja:: rdna1151-gl1c-utilization-gfx1151
         :file: _templates/metrics_table.j2

> GL1C Request Statistics

Reviewer: ```suggestion
GL1C request statistics
```

-----------------------

.. tab-set::

   .. tab-item:: RDNA 3.5 (gfx1151)
      :selected:

      .. jinja:: rdna1151-gl1c-request-statistics-gfx1151
         :file: _templates/metrics_table.j2

> GL1C Cache Performance

Reviewer: ```suggestion
GL1C cache performance
```

----------------------

.. tab-set::

   .. tab-item:: RDNA 3.5 (gfx1151)
      :selected:

      .. jinja:: rdna1151-gl1c-cache-performance-gfx1151
         :file: _templates/metrics_table.j2

> GL1C Stalls

Reviewer: ```suggestion
GL1C stalls
```
